docs/manual/installation/update-contao.de.md

```diff
@@ -143,3 +143,113 @@ $ vendor/bin/contao-console contao:migrate
+auch lokal durchführen. Dies hat den Vorteil, dass du im Gegensatz zur Umgebung bei einem Hoster keine Probleme mit 
+nicht erfüllten Systemanforderungen wie z.B. ungenügendem Arbeitsspeicher bekommst, denn du kannst die entsprechende 
+Konfiguration selbst nach Bedarf anpassen.
+
+
+
+#### Voraussetzungen bei Verwendung der Kommandozeile
+
+Was benötigst du auf deinem Computer?
+
+- ein beliebiges Verzeichnis, in dem du arbeitest (dein Arbeitsverzeichnis)
+- PHP, idealerweise in der gleichen Version, wie sie auf deinem Hosting verwendet wird.  
+- Composer (wir gehen hier davon aus, dass du Composer in deinem Arbeitsverzeichnis installierst)
+- Kopien der `composer.json` und `composer.lock` der Contao-Installation bei deinem Hoster
+
+Was benötigst du _nicht_?
+
+- MySQL
+- Eine lokale Contao-Installation
+
+
+#### Die Aktualisierung durchführen
+
+Kopiere die `composer.json` und `composer.lock` von deinem Hosting in dein Arbeitsverzeichnis.
+Im Wesentlichen machst du dann das Gleiche, wie oben unter
+["Aktualisierung über die Kommandozeile"](#aktualisierung-ueber-die-kommandozeile) beschrieben:
+ 
+Öffne ein Terminal und wechsle in das Arbeitsverzeichnis. Führe dort
+
+```bash
+php composer.phar update
+```
+
+aus. Nach dem erfolgreichen Abschluss der Aktualisierung kopierst du die aktualisierte `composer.lock`
+(und die `composer.json`, falls du dort Änderungen gemacht hast) zurück in die Contao-Installation 
+auf deinem Hosting. 
+
+Danach meldest du dich entweder per `ssh` auf deinem Server (Hosting) an
+
+```bash
+ssh benutzername@example.com
+```
+
+und lässt Composer die aktualisierten Pakete installieren
+
+```bash
+php composer.phar install
+```
+
+oder du verwendest den Contao Manager. Dort wählst du unter Systemwartung den Punkt Composer-Abhängigkeiten, Installer 
+ausführen.
+
+![Paketinstallation mit dem Contao-Manager](/de/installation/images/de/composer-install-mit-dem-contao-manager.png?classes=shadow)
+
+Zum Abschluss musst du noch die Datenbanktabellen aktualisieren. 
+
+
+#### Datenbanktabellen aktualisieren
+
+Öffne das [Contao-Installtool](../contao-installtool/) und überprüfe, ob nach der Aktualisierung Änderungen an deiner 
+Datenbank notwendig sind. Führe gegebenenfalls die vorgeschlagenen Änderungen durch und schließe dann das Installtool.
+
+Anstelle des Contao-Installtools kannst du (ab Contao 4.9) zur Aktualisierung der Datenbanktabellen auf der 
+Kommandozeile das Command 
+```bash
+$ vendor/bin/contao-console contao:migrate
+``` 
+verwenden.
+
+Deine Contao-Installation ist jetzt auf dem neuesten Stand.
+
+
+#### Verschiedene PHP-Versionen
+
+Wenn die lokal verwendete PHP-Version eine andere ist, als bei deinem Hosting, musst du in der `composer.json`
+angeben, welche PHP-Version verwendet werden soll:
+
+```json
+    ...
+    "config": {
+        "platform": {
+            "php": "7.4.99"
+        }
+    },
+    "require": {
+        ...
+```
```

Comment:
```suggestion
    }
```
```

Reply:
@netzarbeiter I do not get, what that change is supposed to correct. 

The suggested change removes the closing ` ``` ` and inroduces a `}`. Do you want to add the `}` as the closing brace of `"require": {` block? The backticks to close the code block would still be required though.

Reply:
looks like the unclosed `}` of the pseudo json code caused the error